ptx/sec_deriv_prodquot.ptx

```diff
@@ -156,15 +156,15 @@
       then do some regrouping as shown.
     </p>
 
-    <aside xml:id="aside-derivative-add-zero" vshift="0">
+    <aside xml:id="aside-derivative-add-zero" vshift="0.9">
       <p>
         Adding <m>0</m> in some clever form is a common mathematical proof technique.
       </p>
     </aside>
     
     <p>
       <md>
-        <mrow>\amp\lzoo{x}{f(x)g(x)} = \lim_{h\to0} \frac{f(x+h)g(x+h)-f(x)g(x)}{h}</mrow>
+        <mrow>\lzoo{x}{f(x)g(x)} \amp= \lim_{h\to0} \frac{f(x+h)g(x+h)-f(x)g(x)}{h}</mrow>
         <mrow>\amp \text{ (now add 0 to the numerator) }</mrow>
```

Comment:
Moving the alignment tab here is going to cause a bunch of overfull hboxes.

That will be a problem if this ends up on an odd-numbered page, I think.
And it means that the equations won't fit in HTML: a horizontal slider will be needed.

Reply:
or maybe even pages, not odd, since there's nothing in the margin there.